src/test/java/com/github/theholywaffle/teamspeak3/commands/FileCommandsTest.java

```diff
@@ -0,0 +1,222 @@
+package com.github.theholywaffle.teamspeak3.commands;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileCommandsTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+  @Test
+  public void ftCreateDir_NullPathException() {
+    thrown.expect(IllegalArgumentException.class);
+    FileCommands.ftCreateDir(null, 001, null);
+  }
+
+  @Test
+  public void ftCreateDir() {
```

Comment:
Have you seen this style of test case before or is it something that they are using already? I feel like it doesn't follow the junit framework of ` Arrange` `Act` and `Assert Result`.

Reply:
No, I just didn't want to unnecessarily break the tests up so much. I've had comments in the past saying there's no need to have tests so long-winded.